src/reader/token.ts

```diff
@@ -199,6 +208,31 @@ export function* tokenize(s: string): Iterable<Token> & Iterator<Token> {
         }
         continue;
       }
+      case '#': {
+        // special dispatch
+
+        const j = i;
+        // one character dispatches
+        inc();
+        inc();
```

Comment:
Why are there 2 `inc()`s here? Is it because we are expecting `;` to come right after?

Reply:
; or t or f, yeah

so if the state is

```
#t blah
^
```

it'll read that # and start reading a dispatch by incrementing twice

```
#t blah
  ^
```

then in this case it was a #t, and returns a bool